pyfranca/franca_lexer.py

```diff
@@ -144,6 +175,28 @@ def t_INTEGER(t):
         t.value = int(t.value)
         return t
 
+
+    # noinspection PyPep8Naming,PyIncorrectDocstring
+    @staticmethod
+    def t_BOOLEAN_VAL(t):
+        # noinspection PySingleQuotedDocstring
+        r"(true|false)"
+        t.value = t.value.strip()
+        if t.value == "true":
+            t.value = True
+        else:
+            t.value = False
+        return t
+
+    # Identifier
```

Comment:
Now I am using the regular exprsssion from pycparser
https://github.com/eliben/pycparser/blob/master/pycparser/c_lexer.py

exponent_part = r"""([eE][-+]?[0-9]+)"""
fractional_constant = r"""([0-9]*\.[0-9]+)|([0-9]+\.)"""
floating_constant = '(((('+fractional_constant+')'+exponent_part+'?)|([0-9]+'+exponent_part+'))[FfLl]?)'